openmdao/docs/_theme/static/searchtools.js

```diff
@@ -0,0 +1,598 @@
+/*
+ * searchtools.js
+ * ~~~~~~~~~~~~~~~~
+ *
+ * Sphinx JavaScript utilities for the full-text search.
+ *
+ * :copyright: Copyright 2007-2019 by the Sphinx team, see AUTHORS.
+ * :license: BSD, see LICENSE for details.
+ *
+ * Modified for OpenMDAO:
+ * - option to include/exclude source docs (default is to exclude)
+ * - if source docs are included, sort them to the end of the results with a separator
+ * - if source docs are included, only include one result for each filename
+ *
+ */
+if (!Scorer) {
+    /**
+     * Simple result scoring code.
+     */
+    var Scorer = {
+        // Implement the following function to further tweak the score for each result
+        // The function takes a result array [filename, title, anchor, descr, score]
+        // and returns the new score.
+
+        score: function(result) {
+            // relevance:  non-source > source > test_suite
+            // modifiers:                 -30      -60
+            filename = result[0];
+            if (filename.indexOf("test_suite") >= 0)
+                return result[4] - 60;
+            else if (filename.startsWith("_srcdocs"))
+                return result[4] - 30;
+            else
+                return result[4];
+        },
+
+        // query matches the full name of an object
+        objNameMatch: 11,
+        // or matches in the last dotted part of the object name
+        objPartialMatch: 6,
+        // Additive scores depending on the priority of the object
+        objPrio: {
+            0: 15,  // used to be importantResults
+            1:  5,  // used to be objectResults
+            2: -5   // used to be unimportantResults
+        },
+        //  Used when the priority is not in the mapping.
+        objPrioDefault: 0,
+
+        // query found in title
+        title: 15,
+        partialTitle: 7,
+        // query found in terms
+        term: 5,
+        partialTerm: 2
+    };
+}
+
+if (!splitQuery) {
+    function splitQuery(query) {
```

Comment:
We talked some today about searching for "cross product" as a single entity. Would it be possible to get that to work by modifying this regex to split on whitespace that isn't enclosed in quotes?

Reply:
added support for quoted multi-word search terms